code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -158,6 +158,12 @@
 /obj/item/ammo_box/magazine/internal/shot/toy/crossbow
 	max_ammo = 5
 
+//crossbow
+/obj/item/ammo_box/magazine/internal/shot/crossbow
+	ammo_type = /obj/item/ammo_casing/caseless/crossbowbolt && /obj/item/ammo_casing/apfsdsbolt

```

Comment:
This operation happens with the same operands and the answer is always TRUE; it's just going to remove the operation and put TRUE. 

It has to be constant folding AFAIK because initial datum var definitions have to be constant. 

But whatever, either way it definitely did not work at all. :p